[2025-07-29T04:32:05.672+0000] {processor.py:161} INFO - Started process (PID=164) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:32:05.674+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:32:05.681+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:32:05.680+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:32:05.694+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:32:05.692+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/ml_pipeline_docker.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/ml_pipeline_docker.py", line 20, in <module>
    run_entire_pipeline = DockerOperator(
                          ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 273, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 882, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: run_pipeline). Invalid arguments were:
**kwargs: {'volumes': ['/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/data:/app/data', '/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/models:/app/models']}
[2025-07-29T04:32:05.695+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:32:05.731+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.066 seconds
[2025-07-29T04:32:36.114+0000] {processor.py:161} INFO - Started process (PID=169) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:32:36.115+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:32:36.118+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:32:36.117+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:32:36.130+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:32:36.127+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/ml_pipeline_docker.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/ml_pipeline_docker.py", line 20, in <module>
    run_entire_pipeline = DockerOperator(
                          ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 273, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 882, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: run_pipeline). Invalid arguments were:
**kwargs: {'volumes': ['/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/data:/app/data', '/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/models:/app/models']}
[2025-07-29T04:32:36.130+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:32:36.149+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.047 seconds
[2025-07-29T04:33:06.596+0000] {processor.py:161} INFO - Started process (PID=174) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:33:06.597+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:33:06.599+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:33:06.599+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:33:06.607+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:33:06.605+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/ml_pipeline_docker.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/ml_pipeline_docker.py", line 20, in <module>
    run_entire_pipeline = DockerOperator(
                          ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 273, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 882, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: run_pipeline). Invalid arguments were:
**kwargs: {'volumes': ['/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/data:/app/data', '/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/models:/app/models']}
[2025-07-29T04:33:06.607+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:33:06.635+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.048 seconds
[2025-07-29T04:33:36.993+0000] {processor.py:161} INFO - Started process (PID=179) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:33:36.995+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:33:37.001+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:33:36.999+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:33:37.010+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:33:37.008+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/ml_pipeline_docker.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/ml_pipeline_docker.py", line 20, in <module>
    run_entire_pipeline = DockerOperator(
                          ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 273, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 882, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: run_pipeline). Invalid arguments were:
**kwargs: {'volumes': ['/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/data:/app/data', '/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/models:/app/models']}
[2025-07-29T04:33:37.011+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:33:37.030+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.047 seconds
[2025-07-29T04:34:07.444+0000] {processor.py:161} INFO - Started process (PID=184) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:34:07.449+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:34:07.452+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:34:07.451+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:34:07.460+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:34:07.458+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/ml_pipeline_docker.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/ml_pipeline_docker.py", line 20, in <module>
    run_entire_pipeline = DockerOperator(
                          ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 273, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 882, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: run_pipeline). Invalid arguments were:
**kwargs: {'volumes': ['/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/data:/app/data', '/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/models:/app/models']}
[2025-07-29T04:34:07.460+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:34:07.486+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.051 seconds
[2025-07-29T04:34:37.915+0000] {processor.py:161} INFO - Started process (PID=189) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:34:37.917+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:34:37.920+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:34:37.919+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:34:37.931+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:34:37.929+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/ml_pipeline_docker.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/ml_pipeline_docker.py", line 20, in <module>
    run_entire_pipeline = DockerOperator(
                          ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 273, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 882, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: run_pipeline). Invalid arguments were:
**kwargs: {'volumes': ['/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/data:/app/data', '/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/models:/app/models']}
[2025-07-29T04:34:37.931+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:34:37.958+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.054 seconds
[2025-07-29T04:35:08.334+0000] {processor.py:161} INFO - Started process (PID=194) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:35:08.335+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:35:08.338+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:35:08.337+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:35:08.350+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:35:08.348+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/ml_pipeline_docker.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/ml_pipeline_docker.py", line 20, in <module>
    run_entire_pipeline = DockerOperator(
                          ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 273, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 882, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: run_pipeline). Invalid arguments were:
**kwargs: {'volumes': ['/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/data:/app/data', '/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/models:/app/models']}
[2025-07-29T04:35:08.350+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:35:08.370+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.046 seconds
[2025-07-29T04:35:38.767+0000] {processor.py:161} INFO - Started process (PID=199) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:35:38.769+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:35:38.771+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:35:38.770+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:35:38.783+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:35:38.781+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/ml_pipeline_docker.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/ml_pipeline_docker.py", line 20, in <module>
    run_entire_pipeline = DockerOperator(
                          ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 273, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 882, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: run_pipeline). Invalid arguments were:
**kwargs: {'volumes': ['/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/data:/app/data', '/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/models:/app/models']}
[2025-07-29T04:35:38.783+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:35:38.804+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.047 seconds
[2025-07-29T04:36:09.192+0000] {processor.py:161} INFO - Started process (PID=204) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:36:09.195+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:36:09.200+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:36:09.200+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:36:09.209+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:36:09.206+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/ml_pipeline_docker.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/ml_pipeline_docker.py", line 20, in <module>
    run_entire_pipeline = DockerOperator(
                          ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 273, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 882, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: run_pipeline). Invalid arguments were:
**kwargs: {'volumes': ['/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/data:/app/data', '/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/models:/app/models']}
[2025-07-29T04:36:09.209+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:36:09.246+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.062 seconds
[2025-07-29T04:36:39.618+0000] {processor.py:161} INFO - Started process (PID=209) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:36:39.619+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:36:39.622+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:36:39.622+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:36:39.632+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:36:39.630+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/ml_pipeline_docker.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/ml_pipeline_docker.py", line 20, in <module>
    run_entire_pipeline = DockerOperator(
                          ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 273, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 882, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: run_pipeline). Invalid arguments were:
**kwargs: {'volumes': ['/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/data:/app/data', '/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/models:/app/models']}
[2025-07-29T04:36:39.633+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:36:39.656+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.047 seconds
[2025-07-29T04:37:10.050+0000] {processor.py:161} INFO - Started process (PID=214) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:37:10.052+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:37:10.055+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:37:10.054+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:37:10.069+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:37:10.066+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/ml_pipeline_docker.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/ml_pipeline_docker.py", line 20, in <module>
    run_entire_pipeline = DockerOperator(
                          ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 273, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 882, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: run_pipeline). Invalid arguments were:
**kwargs: {'volumes': ['/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/data:/app/data', '/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/models:/app/models']}
[2025-07-29T04:37:10.070+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:37:10.103+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.066 seconds
[2025-07-29T04:37:40.490+0000] {processor.py:161} INFO - Started process (PID=219) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:37:40.491+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:37:40.493+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:37:40.493+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:37:40.502+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:37:40.500+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/ml_pipeline_docker.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/ml_pipeline_docker.py", line 20, in <module>
    run_entire_pipeline = DockerOperator(
                          ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 273, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 882, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: run_pipeline). Invalid arguments were:
**kwargs: {'volumes': ['/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/data:/app/data', '/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/models:/app/models']}
[2025-07-29T04:37:40.502+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:37:40.536+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.055 seconds
[2025-07-29T04:38:13.185+0000] {processor.py:161} INFO - Started process (PID=180) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:38:13.188+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:38:13.192+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:38:13.192+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:38:13.212+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:38:13.210+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/ml_pipeline_docker.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/ml_pipeline_docker.py", line 20, in <module>
    run_entire_pipeline = DockerOperator(
                          ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 273, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 882, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: run_pipeline). Invalid arguments were:
**kwargs: {'volumes': ['/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/data:/app/data', '/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/models:/app/models']}
[2025-07-29T04:38:13.213+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:38:13.236+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.058 seconds
[2025-07-29T04:38:43.695+0000] {processor.py:161} INFO - Started process (PID=185) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:38:43.696+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:38:43.702+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:38:43.701+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:38:43.715+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:38:43.711+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/ml_pipeline_docker.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/ml_pipeline_docker.py", line 20, in <module>
    run_entire_pipeline = DockerOperator(
                          ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 273, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 882, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: run_pipeline). Invalid arguments were:
**kwargs: {'volumes': ['/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/data:/app/data', '/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/models:/app/models']}
[2025-07-29T04:38:43.716+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:38:43.743+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.062 seconds
[2025-07-29T04:39:14.117+0000] {processor.py:161} INFO - Started process (PID=190) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:39:14.118+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:39:14.121+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:39:14.120+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:39:14.132+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:39:14.129+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/ml_pipeline_docker.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/ml_pipeline_docker.py", line 20, in <module>
    run_entire_pipeline = DockerOperator(
                          ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 273, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 882, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: run_pipeline). Invalid arguments were:
**kwargs: {'volumes': ['/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/data:/app/data', '/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/models:/app/models']}
[2025-07-29T04:39:14.132+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:39:14.152+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.047 seconds
[2025-07-29T04:39:44.540+0000] {processor.py:161} INFO - Started process (PID=195) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:39:44.542+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:39:44.544+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:39:44.544+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:39:44.555+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:39:44.553+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/ml_pipeline_docker.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/ml_pipeline_docker.py", line 20, in <module>
    run_entire_pipeline = DockerOperator(
                          ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 273, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 882, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: run_pipeline). Invalid arguments were:
**kwargs: {'volumes': ['/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/data:/app/data', '/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/models:/app/models']}
[2025-07-29T04:39:44.556+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:39:44.577+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.044 seconds
[2025-07-29T04:40:15.022+0000] {processor.py:161} INFO - Started process (PID=200) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:40:15.023+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:40:15.026+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:40:15.025+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:40:15.034+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:40:15.032+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/ml_pipeline_docker.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/ml_pipeline_docker.py", line 20, in <module>
    run_entire_pipeline = DockerOperator(
                          ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 273, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 882, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: run_pipeline). Invalid arguments were:
**kwargs: {'volumes': ['/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/data:/app/data', '/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/models:/app/models']}
[2025-07-29T04:40:15.034+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:40:15.059+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.048 seconds
[2025-07-29T04:40:45.464+0000] {processor.py:161} INFO - Started process (PID=205) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:40:45.466+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:40:45.469+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:40:45.468+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:40:45.480+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:40:45.476+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/ml_pipeline_docker.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/ml_pipeline_docker.py", line 20, in <module>
    run_entire_pipeline = DockerOperator(
                          ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 273, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 882, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: run_pipeline). Invalid arguments were:
**kwargs: {'volumes': ['/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/data:/app/data', '/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/models:/app/models']}
[2025-07-29T04:40:45.481+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:40:45.504+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.055 seconds
[2025-07-29T04:41:15.845+0000] {processor.py:161} INFO - Started process (PID=210) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:41:15.847+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:41:15.851+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:41:15.851+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:41:15.859+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:41:15.857+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/ml_pipeline_docker.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/ml_pipeline_docker.py", line 20, in <module>
    run_entire_pipeline = DockerOperator(
                          ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 273, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 882, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: run_pipeline). Invalid arguments were:
**kwargs: {'volumes': ['/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/data:/app/data', '/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/models:/app/models']}
[2025-07-29T04:41:15.859+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:41:15.883+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.049 seconds
[2025-07-29T04:41:46.290+0000] {processor.py:161} INFO - Started process (PID=215) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:41:46.292+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:41:46.298+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:41:46.297+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:41:46.305+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:41:46.303+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/ml_pipeline_docker.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/ml_pipeline_docker.py", line 20, in <module>
    run_entire_pipeline = DockerOperator(
                          ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 273, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 485, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 882, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: run_pipeline). Invalid arguments were:
**kwargs: {'volumes': ['/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/data:/app/data', '/Users/erosestante/Documents/mlops-no2/ac112e6b40ed278155f9ba01f3d16c384cac5674e1cea62f7c67f4cda328a01b_no2_prediction-1/models:/app/models']}
[2025-07-29T04:41:46.306+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:41:46.331+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.049 seconds
[2025-07-29T04:41:48.334+0000] {processor.py:161} INFO - Started process (PID=220) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:41:48.338+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:41:48.342+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:41:48.342+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:41:48.383+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:41:48.581+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:41:48.580+0000] {override.py:1858} INFO - Created Permission View: can read on DAG:ml_pipeline_docker
[2025-07-29T04:41:48.585+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:41:48.585+0000] {override.py:1858} INFO - Created Permission View: can delete on DAG:ml_pipeline_docker
[2025-07-29T04:41:48.588+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:41:48.588+0000] {override.py:1858} INFO - Created Permission View: can edit on DAG:ml_pipeline_docker
[2025-07-29T04:41:48.588+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:41:48.588+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:41:48.594+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:41:48.594+0000] {dag.py:3111} INFO - Creating ORM DAG for ml_pipeline_docker
[2025-07-29T04:41:48.594+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:41:48.594+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:41:48.602+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.276 seconds
[2025-07-29T04:42:05.884+0000] {processor.py:161} INFO - Started process (PID=225) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:42:05.888+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:42:05.891+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:42:05.890+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:42:05.912+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:42:05.933+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:42:05.932+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:42:05.949+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:42:05.949+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:42:05.961+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.095 seconds
[2025-07-29T04:42:17.129+0000] {processor.py:161} INFO - Started process (PID=230) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:42:17.130+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:42:17.136+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:42:17.135+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:42:17.153+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:42:17.175+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:42:17.175+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:42:17.185+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:42:17.185+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:42:17.197+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.078 seconds
[2025-07-29T04:42:47.551+0000] {processor.py:161} INFO - Started process (PID=235) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:42:47.552+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:42:47.555+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:42:47.555+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:42:47.577+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:42:47.605+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:42:47.605+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:42:47.614+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:42:47.614+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:42:47.626+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.086 seconds
[2025-07-29T04:43:18.025+0000] {processor.py:161} INFO - Started process (PID=240) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:43:18.026+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:43:18.029+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:43:18.029+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:43:18.045+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:43:18.075+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:43:18.075+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:43:18.087+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:43:18.087+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:43:18.096+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.081 seconds
[2025-07-29T04:43:32.247+0000] {processor.py:161} INFO - Started process (PID=245) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:43:32.249+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:43:32.252+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:43:32.251+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:43:32.271+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:43:32.308+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:43:32.307+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:43:32.316+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:43:32.316+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:43:32.331+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.093 seconds
[2025-07-29T04:43:47.440+0000] {processor.py:161} INFO - Started process (PID=250) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:43:47.445+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:43:47.448+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:43:47.447+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:43:47.469+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:43:47.501+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:43:47.500+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:43:47.508+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:43:47.508+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:43:47.519+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.088 seconds
[2025-07-29T04:44:26.152+0000] {processor.py:161} INFO - Started process (PID=180) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:44:26.155+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:44:26.159+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:44:26.159+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:44:26.179+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:44:26.215+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:44:26.215+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:44:26.225+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:44:26.224+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:44:26.234+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.087 seconds
[2025-07-29T04:44:56.581+0000] {processor.py:161} INFO - Started process (PID=185) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:44:56.582+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:44:56.588+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:44:56.587+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:44:56.613+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:44:56.653+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:44:56.653+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:44:56.668+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:44:56.668+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:44:56.678+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.109 seconds
[2025-07-29T04:45:26.904+0000] {processor.py:161} INFO - Started process (PID=190) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:45:26.906+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:45:26.909+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:45:26.908+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:45:26.924+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:45:26.952+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:45:26.952+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:45:26.960+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:45:26.960+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:45:26.969+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.073 seconds
[2025-07-29T04:45:57.256+0000] {processor.py:161} INFO - Started process (PID=195) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:45:57.258+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:45:57.260+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:45:57.260+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:45:57.275+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:45:57.308+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:45:57.308+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:45:57.315+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:45:57.315+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:45:57.323+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.079 seconds
[2025-07-29T04:46:27.604+0000] {processor.py:161} INFO - Started process (PID=202) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:46:27.607+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:46:27.614+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:46:27.613+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:46:27.671+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:46:27.811+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:46:27.811+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:46:27.885+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:46:27.885+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:46:27.912+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.327 seconds
[2025-07-29T04:46:58.104+0000] {processor.py:161} INFO - Started process (PID=207) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:46:58.106+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:46:58.108+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:46:58.108+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:46:58.125+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:46:58.166+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:46:58.166+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:46:58.181+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:46:58.181+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:46:58.196+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.102 seconds
[2025-07-29T04:47:28.302+0000] {processor.py:161} INFO - Started process (PID=212) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:47:28.303+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:47:28.305+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:47:28.305+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:47:28.326+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:47:28.359+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:47:28.359+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:47:28.371+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:47:28.371+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:47:28.382+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.090 seconds
[2025-07-29T04:47:58.757+0000] {processor.py:161} INFO - Started process (PID=217) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:47:58.758+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:47:58.761+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:47:58.761+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:47:58.782+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:47:58.819+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:47:58.819+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:47:58.833+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:47:58.833+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:47:58.848+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.106 seconds
[2025-07-29T04:48:29.182+0000] {processor.py:161} INFO - Started process (PID=222) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:48:29.183+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:48:29.187+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:48:29.186+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:48:29.205+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:48:29.243+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:48:29.242+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:48:29.258+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:48:29.258+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:48:29.268+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.094 seconds
[2025-07-29T04:48:59.622+0000] {processor.py:161} INFO - Started process (PID=227) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:48:59.624+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:48:59.627+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:48:59.626+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:48:59.645+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:48:59.679+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:48:59.678+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:48:59.688+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:48:59.688+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:48:59.698+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.087 seconds
[2025-07-29T04:49:29.944+0000] {processor.py:161} INFO - Started process (PID=232) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:49:29.945+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:49:29.950+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:49:29.949+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:49:29.971+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:49:30.012+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:49:30.011+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:49:30.023+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:49:30.023+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:49:30.035+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.100 seconds
[2025-07-29T04:50:00.347+0000] {processor.py:161} INFO - Started process (PID=237) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:50:00.349+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:50:00.351+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:50:00.351+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:50:00.369+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:50:00.413+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:50:00.412+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:50:00.427+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:50:00.426+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:50:00.443+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.108 seconds
[2025-07-29T04:50:30.773+0000] {processor.py:161} INFO - Started process (PID=242) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:50:30.776+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:50:30.779+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:50:30.778+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:50:30.797+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:50:30.833+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:50:30.832+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:50:30.844+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:50:30.844+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:50:30.854+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.090 seconds
[2025-07-29T04:51:01.256+0000] {processor.py:161} INFO - Started process (PID=247) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:51:01.272+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:51:01.275+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:51:01.275+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:51:01.294+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:51:01.331+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:51:01.331+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:51:01.342+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:51:01.342+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:51:01.354+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.110 seconds
[2025-07-29T04:51:31.635+0000] {processor.py:161} INFO - Started process (PID=252) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:51:31.636+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:51:31.643+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:51:31.643+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:51:31.662+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:51:31.699+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:51:31.699+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:51:31.709+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:51:31.709+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:51:31.721+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.094 seconds
[2025-07-29T04:52:02.093+0000] {processor.py:161} INFO - Started process (PID=257) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:52:02.095+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:52:02.098+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:52:02.097+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:52:02.115+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:52:02.157+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:52:02.157+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:52:02.172+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:52:02.172+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:52:02.184+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.103 seconds
[2025-07-29T04:53:02.877+0000] {processor.py:161} INFO - Started process (PID=267) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:53:02.878+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:53:02.881+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:53:02.880+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:53:02.899+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:53:02.948+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:53:02.948+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:53:02.961+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:53:02.961+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:53:02.972+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.108 seconds
[2025-07-29T04:53:33.327+0000] {processor.py:161} INFO - Started process (PID=272) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:53:33.328+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:53:33.331+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:53:33.331+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:53:33.349+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:53:33.383+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:53:33.383+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:53:33.394+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:53:33.394+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:53:33.403+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.090 seconds
[2025-07-29T04:54:03.770+0000] {processor.py:161} INFO - Started process (PID=277) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:54:03.771+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:54:03.776+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:54:03.773+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:54:03.795+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:54:03.831+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:54:03.830+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:54:03.842+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:54:03.842+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:54:03.852+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.093 seconds
[2025-07-29T04:54:34.263+0000] {processor.py:161} INFO - Started process (PID=282) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:54:34.264+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:54:34.267+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:54:34.267+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:54:34.288+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:54:34.328+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:54:34.328+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:54:34.338+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:54:34.338+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:54:34.348+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.093 seconds
[2025-07-29T04:55:04.747+0000] {processor.py:161} INFO - Started process (PID=287) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:55:04.748+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:55:04.751+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:55:04.751+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:55:04.770+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:55:04.806+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:55:04.806+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:55:04.816+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:55:04.816+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:55:04.826+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.091 seconds
[2025-07-29T04:55:35.247+0000] {processor.py:161} INFO - Started process (PID=292) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:55:35.248+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:55:35.254+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:55:35.253+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:55:35.275+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:55:35.328+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:55:35.328+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:55:35.337+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:55:35.337+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:55:35.349+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.115 seconds
[2025-07-29T04:56:05.786+0000] {processor.py:161} INFO - Started process (PID=297) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:56:05.787+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:56:05.790+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:56:05.789+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:56:05.810+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:56:05.845+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:56:05.845+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:56:05.855+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:56:05.855+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:56:05.867+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.091 seconds
[2025-07-29T04:56:36.227+0000] {processor.py:161} INFO - Started process (PID=302) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:56:36.228+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:56:36.231+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:56:36.231+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:56:36.252+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:56:36.294+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:56:36.294+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:56:36.310+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:56:36.310+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:56:36.323+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.107 seconds
[2025-07-29T04:57:06.780+0000] {processor.py:161} INFO - Started process (PID=307) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:57:06.781+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:57:06.784+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:57:06.783+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:57:06.805+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:57:06.840+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:57:06.840+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:57:06.852+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:57:06.852+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:57:06.863+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.094 seconds
[2025-07-29T04:57:37.202+0000] {processor.py:161} INFO - Started process (PID=312) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:57:37.204+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:57:37.212+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:57:37.209+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:57:37.231+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:57:37.276+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:57:37.276+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:57:37.288+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:57:37.288+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:57:37.302+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.110 seconds
[2025-07-29T04:58:07.772+0000] {processor.py:161} INFO - Started process (PID=317) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:58:07.774+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:58:07.778+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:58:07.776+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:58:07.796+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:58:07.842+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:58:07.841+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:58:07.855+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:58:07.854+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:58:07.867+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.104 seconds
[2025-07-29T04:58:38.230+0000] {processor.py:161} INFO - Started process (PID=322) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:58:38.231+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:58:38.235+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:58:38.234+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:58:38.257+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:58:38.297+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:58:38.297+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:58:38.311+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:58:38.310+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:58:38.324+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.113 seconds
[2025-07-29T04:59:08.633+0000] {processor.py:161} INFO - Started process (PID=327) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:59:08.634+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:59:08.637+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:59:08.636+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:59:08.657+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:59:08.694+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:59:08.694+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:59:08.709+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:59:08.709+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:59:08.720+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.098 seconds
[2025-07-29T04:59:39.104+0000] {processor.py:161} INFO - Started process (PID=332) to work on /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:59:39.107+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/ml_pipeline_docker.py for tasks to queue
[2025-07-29T04:59:39.110+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:59:39.110+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:59:39.125+0000] {processor.py:840} INFO - DAG(s) 'ml_pipeline_docker' retrieved from /opt/airflow/dags/ml_pipeline_docker.py
[2025-07-29T04:59:39.158+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:59:39.158+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-07-29T04:59:39.169+0000] {logging_mixin.py:188} INFO - [2025-07-29T04:59:39.169+0000] {dag.py:3947} INFO - Setting next_dagrun for ml_pipeline_docker to None, run_after=None
[2025-07-29T04:59:39.179+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/ml_pipeline_docker.py took 0.085 seconds
